src/components/StorageOverview/Inventory/Inventory.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  DashboardCardTitleHelp,
+} from '../../Dashboard/DashboardCard';
+import { StorageOverviewContext } from '../StorageOverviewContext';
+import { mapNodesToProps, mapPodsToProps, mapPvcsToProps, mapPvsToProps, mapVmsToProps } from './utils';
+import { InventoryRow } from '../../Dashboard/Inventory/InventoryRow';
+
+const InventoryBody = ({ nodes, pods, vms, vmis, pvs, pvcs, migrations }) => (
+  <React.Fragment>
+    <InventoryRow title="Nodes" {...mapNodesToProps(nodes)} />
```

Comment:
Looking at the design - you want `Nodes`, `Disks`, `Pods`, `PVs` and `PVCs`. You are missing `Disks` but I guess you will add them later, however `VMs` here are not needed so you can get rid of `vm`, `vmi` and `migrations`